docs/_instructions/export.md

```diff
@@ -30,7 +30,7 @@ The Exporters use a plugin mechanism and therefore the list is not closed. The f
 
 * `bnd.executablejar.pack` – Exports a JAR file using the launcher from the `-runpath` or the default if no launcher is specified.
 * `bnd.executablejar` – Similar to the previous but does not support profiles, has no automatic bsn assigned, and entries are not signed. The reason there are two types for the more or less the same format with subtle differences is for backward compatibility.
-* `bnd.runbundles` – A JAR with the runbundles
+* `bnd.runbundles` – A JAR with the runbundles. As optional Attributes `targetDir` will output all runbundles in the set directory in the resulting jar and `template` will use a given jar/zip as template to which the bundles will be added.
```

Comment:
template will also support a directory value given how you construct the Jar object from the template.

Reply:
good point! If modified the sentence

Reply:
BTW, while you're here there is an error in this description. bnd.runbundles just drops the bundles into a directory so it should actually read something like: `A directory containing the runbundles. ...`

Reply:
@rotty3000 do you refer to the description for the `template`? If so, the new Version is: `template` will use a given jar/zip or directory as template to which the bundles will be added.

Reply:
No I meant that the bnd.runbundles exporter produces a directory with bundles in it, but the current doc says it creates a jar with bundles in it.

Reply:
Well, yes and no. The `RunBundlesExporter` quite specifically creates a `Jar` and adds all the resources and by default bnd will export write it to a jar, especially when using the `-export` instruction. I guess it works different for maven, but I have no clue how I would need to describe it without confusing any user of the workspace model.

Reply:
odd! you are correct. I was mistaken.